src/Application.cpp

```diff
@@ -181,6 +183,25 @@ void Application::handleStatsPageRequest(AsyncWebServerRequest *request)
   request->send(SPIFFS, stats_file, getContentType(stats_file), false, std::bind(&Application::processStatsPageHTML, this, std::placeholders::_1));
 }
 
+void Application::handleScriptRequest(AsyncWebServerRequest *request)
+{
+  String script_file = "/script.js";
+  request->send(SPIFFS, script_file, getContentType(script_file), false, std::bind(&Application::processScriptFile, this, std::placeholders::_1));
+}
```

Comment:
This function is missing the log line (`Serial.print(...)`), which strictly speaking isn't really needed but all the other handlers do log their web requests.